docs/jobs-site.md

```diff
@@ -117,16 +117,35 @@ Each job will either succeed or fail. In either case, the output and log files a
 
 ## Viewing analysis outputs on the server
 
-You can view `moderately_sensitive` outputs from any of your submitted _jobs_ via the Jobs website **if you have access to and are logged into the backend the job was run on**.
+You can view `moderately_sensitive` outputs from any of your submitted _jobs_
+via the Jobs website **if you have access to and are logged into the backend
+the job was run on**.
 
-Once you are logged into the server:
+However, whilst normally you log into [jobs.opensafely.org] from your machines
+browser using Github, the secure server does not have access to
+Github. So you need to use an alternate method to login, by generating a Single
+Use Token, and then using it on the secure server to log in.
+
+To generate a Single Use Token, before logging on the backend, visit
```

Comment:
`before logging on the backend` -> `before logging *in* on the backend`, PR incoming